packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -815,7 +815,9 @@ class CupertinoTextField extends StatefulWidget {
     BuildContext context,
     EditableTextState editableTextState,
   ) {
-    if (defaultTargetPlatform == TargetPlatform.iOS && SystemContextMenu.isSupported(context)) {
+    if (defaultTargetPlatform == TargetPlatform.iOS &&
```

Comment:
I wonder if we should share this logic somewhere, maybe under `EditableTextState` as `supportsSystemContextMenu`? Not required for this PR but just a thought I had since I see the same logic duplicated across multiple files.